skglm/utils/prox_funcs.py

```diff
@@ -204,3 +204,81 @@ def prox_SLOPE(z, alphas):
             x[i] = d
 
     return x
+
+
+@njit
+def log_1p_exp_vec(x, eps):
```

Comment:
@Badr-MOUFAD good catch

the name is also wrong, but the function seems to be used only one, so we should be able to inline the call and spare the function definition

Reply:
agree :+1: